sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TracezZPageHandler.java

```diff
@@ -522,7 +522,8 @@ private void emitHtmlBody(Map<String, String> queryMap, PrintStream out) {
   }
 
   @Override
-  public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
+  public void emitHtml(
+      String requestMethod, Map<String, String> queryMap, OutputStream outputStream) {
```

Comment:
if the requestMethod isn't used in this method, why pass it in?

Reply:
hmm. I see...it's added to the interface. This make me think the interface isn't quite right, since only the one page needs to know that it's a post vs. get.

Reply:
I'm not sure if any future zPages will require `requestMethod`, but this will also allow any potential new zPage to have access to the requestMethod as well

Reply:
all the more reason to not pollute all the other page implementations with the request method.